libs/core/execution_base/include/hpx/execution_base/operation_state.hpp

```diff
@@ -0,0 +1,98 @@
+//  Copyright (c) 2020 Thomas Heller
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#pragma once
+
+#include <hpx/config/constexpr.hpp>
+#include <hpx/functional/tag_fallback_invoke.hpp>
+#include <hpx/functional/tag_invoke.hpp>
+#include <hpx/functional/traits/is_invocable.hpp>
+
+#include <type_traits>
+#include <utility>
+
+namespace hpx { namespace execution_base { namespace experimental {
```

Comment:
```suggestion
namespace hpx { namespace execution { namespace experimental {
```
? I'll happily do this change since! :D I wasn't very explicit when I first suggested the namespace change.